tests/unit/plugins/module_utils/test_vm.py

```diff
@@ -656,7 +656,7 @@ def test_post_vm_payload_cloud_init_present(self, rest_client):
             "dom": {
                 "blockDevs": [],
                 "bootDevices": [],
-                "CloudInitData": {
+                "CloudIinitData": {
```

Comment:
@domendobnikar typo here?

Reply:
It was a revert to a bug that @tjazsch found, but it's a larger fix that requires integration tests and unit test update, so I made a new [issue ](https://github.com/ScaleComputing/HyperCoreAnsibleCollection/issues/41)rather than fixing it here, since it is unrelated.